### PR TITLE
Cleanup UI

### PR DESCRIPTION
**Pull request by wowtor**
This PR splits MainAcitivity in three parts:

* Terms and conditions are moved to AppInfoAcitivity
* Buttons and switches moved to PreferencesActivity
* Measurements remain in MainActivity, which is renamed to ViewMeasurementsActivity

The diff is quite big but most code is just moved to another file. No significant changes in behavior.

### CODE REVIEW

File: app/src/main/java/nl/nfi/cellscanner/ViewMeasurementsActivity.java [hunk 1]
```diff
@@ -45,14 +45,14 @@
 import static nl.nfi.cellscanner.recorder.RecorderUtils.inRecordingState;
```
Comment (prinsniels) [inline app/src/main/java/nl/nfi/cellscanner/ViewMeasurementsActivity.java:49]: please remove unused imports

File: app/src/main/java/nl/nfi/cellscanner/PreferencesActivity.java [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+
+    private PreferenceFragment prefs;
+
+    private SwitchPreferenceCompat swRecordingMaster;
+    private SwitchPreferenceCompat swGPSRecord;
+    private SwitchPreferenceCompat swGPSPrecision;
+
+    public static class PreferenceFragment extends PreferenceFragmentCompat
+    {
+        private final PreferencesActivity a;
```
Comment (prinsniels) [inline app/src/main/java/nl/nfi/cellscanner/PreferencesActivity.java]: a => preferencesActivity

File: app/src/main/java/nl/nfi/cellscanner/PreferencesActivity.java [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+                    return true;
+                }
+            });
+        }
+    }
+
+    private void setupRecording() {
+        swRecordingMaster = prefs.findPreference("APP_RECORDING");
+        swGPSRecord = prefs.findPreference("GPS_RECORDING");
+        swGPSPrecision = prefs.findPreference("gps_high_accuracy");
```
Comment (prinsniels) [inline app/src/main/java/nl/nfi/cellscanner/PreferencesActivity.java]: ALL CAPS please =D

File: app/src/main/res/xml/preferences.xml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            app:defaultValue="false"/>
+
+        <SwitchPreferenceCompat
+            app:key="GPS_RECORDING"
+            app:title="Record locations"
+            app:summary="Enable location logging"
+            app:defaultValue="true"/>
+
+        <SwitchPreferenceCompat
+            app:key="gps_high_accuracy"
```
Comment (prinsniels) [inline app/src/main/res/xml/preferences.xml]: All keys in ALL CAPS please

File: app/src/main/java/nl/nfi/cellscanner/PreferencesActivity.java [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+
+        swGPSRecord.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener(){
+            @Override
+            public boolean onPreferenceChange(Preference preference, Object newValue) {
+                swGPSPrecision.setEnabled((boolean)newValue);
+                return true;
+            }
+        });
+
+        swGPSPrecision.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener(){
```
Comment (prinsniels) [inline app/src/main/java/nl/nfi/cellscanner/PreferencesActivity.java:162]: Switch is no longer working on device. User is not able to toggle the switch

Comment (prinsniels) [reply app/src/main/java/nl/nfi/cellscanner/PreferencesActivity.java:162]: possible cause: Switch is bound to GPS_HIGH_PRECISION_RECORDING in on toggle

**Review by prinsniels — CHANGES_REQUESTED** [review]
High Accuracy GPS recording is broken

File: app/src/main/java/nl/nfi/cellscanner/PreferencesActivity.java (hunk 5 above)
Comment (wowtor) [reply app/src/main/java/nl/nfi/cellscanner/PreferencesActivity.java:162]: fixed

File: app/src/main/res/xml/preferences.xml (hunk 4 above)
Comment (wowtor) [reply app/src/main/res/xml/preferences.xml]: fixed

**Pull request merged**